examples/plot_otda_laplacian.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+"""
+========================
+OT for domain adaptation
```

Comment:
add proper titke since it sill appear in the documentation

Reply:
done